### PR TITLE
Cypress test optimization.

### PR DESCRIPTION
**Pull request by dvkruchinin**
Optimize tests to speed up passage.
Most tests use one common annotation task. Search by task name. If not exist - creation.
Redesigned the plugin generate the image. (The time to generate 30 images is just over 1 second.)
Redesigned the logic of job search in multi job task. The starting point is always set to the Frame column with a value of "0-".
Adaptation of tests.

### Motivation and context
At the moment, the passage of the cypress test takes about 10-12 minutes. Here the time takes about 5-7 minutes.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/support/const.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    typeAttribute: 'Text'
+}
+
+it('Prepare to testing', () => {
+    cy.visit('/')
+    cy.login()
+    cy.get('.cvat-tasks-page').should('exist')
+    let listItems = []
+    cy.document().then((doc) => {
+        const collection = Array.from(doc.querySelectorAll('.cvat-text-color'))
```
[the remark this answers is not in the document]
Comment (bsekachev) [reply tests/cypress/support/const.js:44]: Here #2344

[the remark this answers is not in the document]
Comment (dvkruchinin) [reply tests/cypress/support/const.js:44]: OK. Thanks. Will update after merge.

**Comment by dvkruchinin** [comment]
Need to increase the number of images and the segment Size parameter. This will be useful for future test scenarios.

**Comment by bsekachev** [comment]
> Need to increase the number of images and the segment Size parameter. This will be useful for future test scenarios.

If you think so, you are welcome. 
Let me know when the PR is ready

**Comment by dvkruchinin** [comment]
@bsekachev PR is ready for a review.
About getting the label name when creating an object. When I investigate the issue, the work was not the easiest. I think it would be better to put it in a separate task.

**Comment by bsekachev** [comment]
> When I investigate the issue, the work was not the easiest. I think it would be better to put it in a separate task.

Okay, let's consider it like a different task. Could you formulate it and sent to our mail thread?

**Pull request merged**